core/src/test/java/org/apache/iceberg/TestSplitPlanning.java

```diff
@@ -63,21 +63,21 @@ public void setupTable() throws IOException {
 
   @Test
   public void testBasicSplitPlanning() {
-    List<DataFile> files128MB = newFiles(4, 128 * 1024 * 1024);
-    appendFiles(files128MB);
+    List<DataFile> files128Mb = newFiles(4, 128 * 1024 * 1024);
```

Comment:
Nit: Should MB and KB be added to a white list somewhere?

Reply:
Don't necessarily think so. Mb and Kb are as well understood.